Program.cs

```diff
@@ -20,5 +20,10 @@ public static void Main(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>();
+
+        public static void Fake(string[])
+        {
```

Comment:
```suggestion
        {
         dsadsadsad
         sadsadsadsa
         dasdsada
         }
```
How about this awesome code ?